src/build.py

```diff
@@ -1397,7 +1406,7 @@ def TestBare():
       wasmjs=os.path.join(INSTALL_LIB, 'wasm.js'),
       extra_files=[os.path.join(INSTALL_LIB, 'musl.wasm')])
 
-  if IsMac():
+  if IsMac() and 'JSC' not in buildbot.WarnedList():
```

Comment:
I added `DidStepFailOrWarn()`, because `DidStepFail` sounds like it might not include warnings.